recipes/parallelproj/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "parallelproj" %}
+{% set version = "1.2.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gschramm/parallelproj/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 826a83e51db6459004d17c030c21fb32e7cb5205a45c28c8f50f87f235321196
+
+build:
+  number: {{ build_number }}
```

```suggestion
  number: {{ build_number }}
  ignore_run_exports:
    - cudatoolkit
```
It looks like [you aren't linking to the cudatoolkit](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=619558&view=logs&j=eb6ecf2b-0363-5c1a-baa7-2d771889d197&t=d8576026-8998-52ff-d027-1d631b0fb837&l=1257), so it's not needed.